app/lib/constants.js

```diff
@@ -4,5 +4,6 @@ module.exports = {
     PERIOD_END: 'PeriodEnd',
     SUPPLIER: 'Supplier',
   },
-  OUTPUT_DIR: 'site/json',
+  OUTPUT_DIR: 'output',
+  CONTAINER_NAME: process.env.CONTAINER_NAME || 'etl-output'
```

Comment:
Using the same container name as that used for `pomi-data-etl` has the potential for the 2 independent ETL processes to use the same file and overwrite each other's output. Perhaps this should be a different container? Or a folder within the container? Ideally the change would also be replicated within the other ETL project.